pkg/core/server/registry_bridge.go

```diff
@@ -306,6 +306,24 @@ func (s *Server) registerSelfOnEth() error {
 	return nil
 }
 
+func (s *Server) validateValidatorRegistration(tx *core_proto.SignedTransaction_ValidatorRegistration) error {
+	db := s.db
+
+	cometAddr := tx.ValidatorRegistration.CometAddress
+
+	node, err := db.GetRegisteredNodeByCometAddress(context.Background(), cometAddr)
+	if err != nil {
+		return fmt.Errorf("validation db fail: %v", err)
+	}
```

Comment:
Unregistered nodes attempting to send a registration tx will get blocked by this, won't they